lib/rosstalk.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of the Companion project
+ * Copyright (c) 2018 Bitfocus AS
+ * Authors: William Viker <william@bitfocus.io>, Håkon Nessjøen <haakon@bitfocus.io>
+ *
+ * This program is free software.
+ * You should have received a copy of the MIT licence as well as the Bitfocus
+ * Individual Contributor License Agreement for companion along with
+ * this program.
+ *
+ * You can be released from the requirements of the license by purchasing
+ * a commercial license. Buying such a license is mandatory as soon as you
+ * develop commercial activities involving the Companion software without
+ * disclosing the source code of your own applications.
+ *
+ */
+
+let net = require('net');
+
+class rosstalk {
+	constructor(_system) {
+		this.system = _system;
+		this.active = false;
+		this.socket = this.createSocket();
+		this.port   = 7788;
+		this.release_time = 20; // ms to send button release
+
+		this.system.emit('get_userconfig', obj => {
+			this.config = obj;
+
+			if(this.config['rosstalk_enabled'] === undefined) {
+				this.config['rosstalk_enabled'] = false;
+				this.system.emit('set_userconfig_key', 'rosstalk_enabled', this.config['rosstalk_enabled']);
+			}
+
+			if(this.config['rosstalk_enabled'] === true) {
+				try {
+					this.listen();
+				} catch(e) {
+					console.log('Error listening for artnet', e);
+				}
+			}
+		});
+
+		this.system.on('set_userconfig_key', (key, val) => {
+			if(key !== 'rosstalk_enabled') return;
```

Comment:
This should have curly braces

Reply:
There should be a space after the «if» codeword also